packages/server/providers/pouchdb.js

```diff
@@ -181,6 +181,36 @@ const pouchDb = (app, io, pouchOptions) => {
       .catch(() => res.send([]))
   })
 
+  app.get('/:wargame/turns', (req, res) => {
+    const databaseName = checkSqliteExists(req.params.wargame)
+
+    if (!databaseName) {
+      res.status(404).send({ msg: 'Wrong Wargame Name', data: null })
+    }
+
+    const db = new PouchDB(databaseName, pouchOptions)
+
+    db.find({
```

Comment:
@lilitkarapetyan - can you remind me why we do `not-equal` for `wargameSettings`?  At the start of the game, this is a valid description of game time and game turn length.

Reply:
Hello Ian, I will add it.